hpat/tests/test_series.py

```diff
@@ -266,6 +266,54 @@ def test_impl(A):
         A = pd.Series(np.random.ranf(n))
         pd.testing.assert_series_equal(hpat_func(A), test_impl(A))
 
+    def test_series_argsort2(self):
+        def test_impl(S):
+            return S.argsort()
+        hpat_func = hpat.jit(test_impl)
+
+        S = pd.Series([5, np.nan, 3, 3, np.nan])
+        pd.testing.assert_series_equal(test_impl(S), hpat_func(S))
+
+    def test_series_argsort_noidx(self):
+        def test_impl(S):
+            return S.argsort()
+
+        hpat_func = hpat.jit(test_impl)
+
+        data_test = [[6, 6, 2, 1, 3, 3, 2, 1, 2],
```

Comment:
I propose to use globally defined data as input data in tests: f2435b0#diff-deca39d332649cea819383154a5d2cb3R39-R62